examples/QmlBarcodeGenerator/cmake/Locations.cmake

```diff
@@ -3,23 +3,21 @@ set(COPYRIGHT "Copyright (c) 2022 Scythe Studio. All rights reserved.")
 set(IDENTIFIER "com.scythestudio.scodes.example")
```

Comment:
This should be updated for the preamble from the open-source license used in project